### PR TITLE
Add angular-elevatezoom-plus w/ git auto-update via single package.json

### LINKED ISSUE #10384: [Request] Add angular-elevatezoom-plus
**Library name:** angular-elevatezoom-plus
**Git repository url:https://github.com/igorlino/angular-elevatezoom-plus**
**npm package url(optional):https://www.npmjs.com/package/angular-ez-plus** 
**License(s):The MIT License (MIT)**
**Official homepage:https://igorlino.github.io/angular-elevatezoom-plus/#/main**
**Wanna say something? Leave message here:plz**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #10384, cc @igorlino

Pull request for issue: #10384 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/igorlino/angular-elevatezoom-plus
 * Official website (optional, not the repository): https://igorlino.github.io/angular-elevatezoom-plus/
 * NPM package url (optional): https://www.npmjs.com/package/angular-ez-plus
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 6
   - Count of stars: 20
   - Count of forks: 11
 * NPM download stats (optional):
   - Downloads in the last day: 12
   - Downloads in the last week: 69
   - Downloads in the last month: 250

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/angular-elevatezoom-plus/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "angular-elevatezoom-plus",
+  "filename": "angular-ezplus.min.js",
```
Comment (sufuf3) [inline ajax/libs/angular-elevatezoom-plus/package.json]: Let's revise to `js/angular-ezplus.min.js`
Thanks.

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. FYR

**Comment by sashberd** [comment]
@PeterDaveHello  What do you mean by confirm. Should I add them manually?

**Comment by PeterDaveHello** [comment]
@sashberd Looks like some of the files are not useful

**Comment by sashberd** [comment]
@PeterDaveHello Are you talking about 
ezplus.directive.js,	
ezplus.module.js in version 1.0.3 ~ 1.1.0?

or about 

widget-ezplus-directive.js 

in versions 1.0.0 ~1.0.2?

All those files are good angular files.. What is your hesitation?

**Comment by sashberd** [comment]
@PeterDaveHello Waiting your answer

**Comment by sashberd** [comment]
@PeterDaveHello ping

**Pull request merged**